test/tests/selftest/pbs_syslog_tests.py

```diff
@@ -46,8 +46,33 @@ class TestSyslog(TestSelf):
     du = DshUtils()
     lu = PBSLogUtils()
 
-    def setup_syslog(self, hostname=None, local_log=1,
-                     syslog_facility=1, syslog_severity=9):
+    severity = {
+                "emerg": 0,
+                "alert": 1,
+                "crit": 2,
+                "err": 3,
+                "warning": 4,
+                "notice": 5,
+                "info": 6,
+                "debug": 7
+               }
+
+    facility = {
+                "No_SYLOG": 0,
+                "LOG_DAEMON": 1,
+                "LOG_LOCAL0": 2,
+                "LOG_LOCAL1": 3,
+                "LOG_LOCAL2": 4,
+                "LOG_LOCAL3": 5,
+                "LOG_LOCAL4": 6,
+                "LOG_LOCAL5": 7,
+                "LOG_LOCAL6": 8,
+                "LOG_LOCAL7": 9
+               }
+
+    def setup_syslog(self, hostname=None, local_log=0,
```

Comment:
it's dangerous to use mutable types in default arguments. Please make the default None and set them to facility's values inside the function if they are None.

Reply:
on second thought, these are just int(s), so it should be fine, sorry about that.